packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -264,6 +264,17 @@ class FlutterCommandRunner extends CommandRunner {
       .toList();
   }
 
+  /// Get the entry-points we want to analyze in the Flutter repo.
+  List<Directory> getRepoAnalysisEntryPoints() {
```

Comment:
We may want to re-write `getRepoPackages()` (above) in terms of these roots rather than flutterRoot.